### ISSUE TITLE
Dashboard accessibility test fails intermittently

### ISSUE THREAD

**Issue by alexbrand** [apprenda/kismatic#410]
We have seen a couple of these failures:
```
Using an upgraded cluster
should allow adding a new storage volume: PASSED
should allow adding a worker node: PASSED
should be able to deploy a workload with ingress: PASSED
should have an accessible dashboard: FAILED: exit status 1
should still be a highly available cluster after upgrade: PASSED
```
